sdk/ml/azure-ai-ml/azure/ai/ml/entities/_component/parallel_component.py

```diff
@@ -86,6 +92,7 @@ def __init__(
         mini_batch_error_threshold: int = None,
         task: ParallelTask = None,
         mini_batch_size: str = None,
+        partition_keys: List = None,
```

Comment:
Could we support serialized str here also? it will require extra logic if user dump the field in CLI, then try to copy it here